src/item.cpp

```diff
@@ -5615,13 +5746,48 @@ layer_level item::get_layer() const
     }
 }
 
-int item::get_coverage() const
+int item::get_avg_coverage() const
 {
     const islot_armor *t = find_armor_data();
-    if( t == nullptr ) {
+    if( !t ) {
+        return 0;
+    }
+    int avg_coverage = 0;
+    int avg_ctr = 0;
+    for( const armor_portion_data &entry : t->data ) {
+        if( entry.covers.has_value() ) {
+            avg_coverage += entry.coverage;
+            ++avg_ctr;
```

Comment:
I think this should increase `avg_ctr` by the number of bodyparts in this portion, rather than 1.